### PR TITLE
feat: added the background gradient into an image

### LINKED ISSUE #14: Section Graident to background image
See this page:

https://ihsvirtualexperience.com/stewardship-series/

### PR DESCRIPTION
**Pull request by lwintch**
added a custom block that allows for a background image at the bottom of a section to have a linear gradient at the top.

Fix #14

Test URLs:
- After: https://feat-section-background-gradient--takeda-ihs--hlxsites.hlx.page/drafts/lwintch/section-background-gradient

### CODE REVIEW

File: blocks/section-background-image-gradient/section-background-image-gradient.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.section-background-image-gradient > div:first-of-type {
+    position: absolute;
+    height: 200px;
+    width: 100%;
+    z-index: 1;
+    background: linear-gradient(
+            180deg,
+            rgb(235 237 238 / 100%) 0%,
```
Comment (lwintch) [inline blocks/section-background-image-gradient/section-background-image-gradient.css:18]: @bstopp, let me know if you'd like me to match these colors with one of the gray color values we already have, and I can translate that and assign to a variable if needed.

**Comment by lwintch** [comment]
After this PR is merged, I'll update the style guide.

**Comment by lwintch** [comment]
> This doesn't quite meet the experience of the live site. The initial background starting color is a bit off, and it's not behind the section in which it is in.
> 
> Here's [the live page](https://ihsvirtualexperience.com/stewardship-series/).
> 
> Here's the[ current version](https://feat-section-background-gradient--takeda-ihs--hlxsites.hlx.page/drafts/stopp/gradient-background-image) using this PR.
> 
> You may have to look into using a Section Metadata attribute like `Background Image`, and then adding pseudo elements or auto-boxing to get it to match.

I did notice this, however I thought the issue was limited to the gradient into the image.  I can make this match the whole section with the angled gradient that would take into the color variation, or I can use a color picker to match the color for each graphic as there are some variations such as the gradient into an image on the first page.

@bstopp should I take into account the whole section with the angle or just the gradient into the picture?

**Comment by bstopp** [comment]
Let me know if you'd rather use #105

**Pull request closed without merging**